tests/e2e/run-test-e2e

```diff
@@ -269,12 +269,14 @@ END_TIME=$(date +%s)
 DURATION=$((END_TIME - START_TIME))
 
 # Calculate minutes and seconds
+DAYS=$((DURATION / 86400))
+HOURS=$(( (DURATION % 86400) / 3600 ))
```

Comment:
shouldn't you also update minutes and seconds then?

Reply:
thanks @sandrobonazzola good catch!